content/04.use-cases-b.md

```diff
@@ -37,36 +37,34 @@ A CC BY 4.0 license specifies that any code (or other creative products) must be
 
+By enabling more comprehensive remote collaboration, GitHub encourages the exchange of ideas among researchers at different institutions and in different countries, which can serve to improve the quality of the research itself by providing open access to data and code.
 
-### Writing a manuscript
+### Writing a manuscript{#writing-manuscript}
 
-<!--*contributors to this section:* Emma J. Hudgins, Katherine Hébert-->
+<!--*contributors to this section:* Emma J. Hudgins, Katherine Hébert, PHPB, Eric-->
 
 Beyond supporting collaborative code development, GitHub can be used for writing manuscripts.
 Writing a manuscript and storing its associated data and code in GitHub increases scientific reproducibility because text, code, and data can be found in one place.
-Writing a manuscript on GitHub may take more time when compared to using conventional text processors [@doi:10.1186/1751-0473-8-7].
-Nevertheless, GitHub has many features that can allow for a powerful collaborative workflow when writing manuscripts.
+Although more time-consuming when compared to using conventional text processors, GitHub has many features that can allow for a powerful collaborative workflow when writing manuscripts [@doi:10.1186/1751-0473-8-7].
 Text documents stored and versioned in GitHub can be instantly displayed when written in Markdown, a lightweight markup language increasingly popular among scientists. 
 When manuscripts are written in Markdown on GitHub, co-authors can contribute changes or suggest revisions to a manuscript in the same ways they would contribute changes or suggest revisions to code (see [Collaborative and asynchronous code editing](#code-editing)).
 Relevant literature or issues can be suggested using the Discussions and Issues features.
 
 Incorporating GitHub into the process of writing a manuscript does not necessarily mean pivoting to an entirely new workflow. 
-For instance, authors who prefer writing in LaTeX can link their new or existing Overleaf projects to GitHub via the Git-Bridge and GitHub Synchronization features ([Table 1](#tbl:compare)), which are available through premium subscriptions to Overleaf or through many institutions worldwide via the Overleaf Commons service (<https://www.overleaf.com/for/universities>) [@url:https://www.overleaf.com/learn/how-to/Using_Git_and_GitHub]. 
-Similar GitHub integrations are available for projects stored in DropBox, Google Drive, among other popular tools that are likely to be familiar to many scientists and their collaborators ([Table 1](#tbl:compare)).
+For instance, authors who prefer writing in LaTeX or Markdown can link their new or existing projects to GitHub via the Git bridge and GitHub Synchronization features ([Table 1](#tbl:compare)), which are available through premium subscriptions and university partnerships to Overleaf   [@url:https://www.overleaf.com/learn/how-to/Using_Git_and_GitHub], through HackMD (<https://hackmd.io>), and others. 
```

Comment:
Feels like maybe too much detail.  Might be enough to mention that both Overleaf and HackMD have ways of synchronizing with GitHub
```suggestion
For instance, authors who prefer writing in LaTeX or Markdown can synchronize Overleaf,  [@url:https://www.overleaf.com/learn/how-to/Using_Git_and_GitHub], HackMD (<https://hackmd.io>), and other platforms directly with a GitHub repository.
```

Reply:
this looks like a better level of detail to me